packages/tpetra/core/src/Tpetra_Details_normImpl.hpp

```diff
@@ -147,6 +147,7 @@ lclNormImpl (const RV& normsOut,
 
   if (lclNumRows == 0) {
     const mag_type zeroMag = Kokkos::ArithTraits<mag_type>::zero ();
+    // DEEP_COPY REVIEW - VALUE-TO-DEVICE
```

Comment:
Untested, used by tpetra_multivector.  The obvious answer would to be to use multiVector to establish its own norm Impl then the identity (kokkos::arithtraits<T>::zero()) wouldn't be necessary.

Reply:
Tested in TpetraCore_MultiVector_UnitTests_MPI_4